index.html

```diff
@@ -184,30 +184,19 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
 <script>
 var fireballSize = 22;
 var getFireballSpeed = function (left) {
-  if (left) {
-    return 5;
-  }
-  else {
-    return 2;
-  }
+  return left ? 5 : 2;
 }
 var wizardSpeed = 3;
 var wizardWidth = 70;
 var getWizardHeight = function () {
   return 1.337 * wizardWidth;
 }
 function getWizardX (width) {
-  return width / 2 - wizardWidth / 2;
+  return (width - wizardWidth) / 2;
 }
 function getWizardY (height) {
-  return height * 2 / 3 - getWizardHeight() / 2;
-  // оставила деление высоты мага на 2 исключительно из-за эстетических соображений
+  return height * 2 / 3 - getWizardHeight();
```

Comment:
в этом случае под ногами мага остается 1/3 высоты. а нам нужно 2/3. поэтому будет вот так
```suggestion
  return height/3 - getWizardHeight();
```